src/components/WalletDropdown/DefaultMenu.tsx

```diff
@@ -120,19 +133,30 @@ const WalletDropdown = ({ setMenu }: { setMenu: (state: MenuState) => void }) =>
       )}
       <Divider />
       {isAuthenticated && (
-        <ToggleMenuItem data-testid="wallet-transactions" onClick={() => setMenu(MenuState.TRANSACTIONS)}>
-          <DefaultText>
-            <Trans>Transactions</Trans>{' '}
-            {pendingTransactions.length > 0 && (
-              <PendingBadge>
-                {pendingTransactions.length} <Trans>Pending</Trans>
-              </PendingBadge>
-            )}
-          </DefaultText>
-          <IconWrap>
-            <ChevronRight size={16} strokeWidth={3} />
-          </IconWrap>
-        </ToggleMenuItem>
+        <div>
```

Comment:
is the `div` needed, or could it be <></>?

Reply:
done ty